### PR TITLE
Bumping cloud-provider-aws to version 1.27

### PR DESCRIPTION
**Pull request by junshun**
Bumping version to get some updated depedencies

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: projects/kubernetes/cloud-provider-aws/expected_artifacts [hunk 1]
```diff
@@ -4,10 +4,10 @@ SHA256SUM.sha512
 SHA512SUM
 SHA512SUM.sha256
 SHA512SUM.sha512
-cloud-provider-aws-linux-amd64-v1.26.1.tar.gz
-cloud-provider-aws-linux-amd64-v1.26.1.tar.gz.sha256
-cloud-provider-aws-linux-amd64-v1.26.1.tar.gz.sha512
-cloud-provider-aws-linux-arm64-v1.26.1.tar.gz
-cloud-provider-aws-linux-arm64-v1.26.1.tar.gz.sha256
-cloud-provider-aws-linux-arm64-v1.26.1.tar.gz.sha512
+cloud-provider-aws-linux-amd64-v1.27.0.tar.gz
```
Comment (jaxesn) [inline projects/kubernetes/cloud-provider-aws/expected_artifacts:7]: You shouldn't need the version in this file. Check one of the others I believe you can use GIT_TAG

**Comment by jaxesn** [comment]
I wonder about this one, projects like this are usually tied to kube versions. Im curious does this latest version work for 1.22+ clusters?

File: projects/kubernetes/cloud-provider-aws/expected_artifacts (hunk 1 above)
Comment (junshun) [reply projects/kubernetes/cloud-provider-aws/expected_artifacts]: done

**Comment by jaxesn** [comment]
Ah yeah that makes any, thanks!

**Review by jaxesn — APPROVED** [review]

**Pull request merged**